ansys/dpf/core/collection.py

```diff
@@ -41,11 +41,6 @@ class Collection:
         default is ``None``, in which case an attempt is made to use the global
         server.
 
-    Examples
```

Comment:
@cbellot000 Why was this one problematic?

Reply:
@PProfizi it's not a supported behavior we should use the implementation instead (MeshesContainer here)